gridstatus/caiso.py

```diff
@@ -1460,19 +1460,19 @@ def _make_timestamp(time_str, today, timezone="US/Pacific"):
     return ts
 
 
-def _get_historical(file, date, verbose=False):
-    try:
-        date_str = date.strftime("%Y%m%d")
-        url = _HISTORY_BASE + "/%s/%s.csv" % (date_str, file)
-        msg = f"Fetching URL: {url}"
-        log(msg, verbose)
-    except Exception:
-        # fallback if today and no historical file yet
-        if utils.is_today(date, CAISO.default_timezone):
-            url = _BASE + "/%s.csv" % file
-            msg = f"Fetching URL: {url}"
-            log(msg, verbose)
-
+def _get_historical(
+    file: str,
```

Comment:
thanks for adding typing. i'd love to do it across more of the codebase as we have time!

Reply:
Sounds good - for now I'll make a habit to add it to functions as I touch them.